src/components/ReadingProgressPage/ReadingHistory/DaysCalendar.tsx

```diff
@@ -0,0 +1,169 @@
+import { useContext, useMemo } from 'react';
```

Comment:
Nothing actionable now but I Calendar component should be extracted to a standalone component so it can be re-used later for other features.